### PR TITLE
completed react-validated-input-component exercise without extras yet.

### PR DESCRIPTION
**Pull request by joseph-w-valdez**
https://user-images.githubusercontent.com/117682160/217398906-e227ac2a-486e-4f5b-88aa-0ee2f4075404.mp4

### CODE REVIEW

File: react-validated-input-component/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+const container = document.querySelector('#root');
+const root = ReactDOM.createRoot(container);
+
+const RegistrationForm = () => {
+  const [password, setPassword] = useState('');
+
+  const isValid = password.length >= 8;
+
+  let warning;
```
Comment (RobertGardner) [inline react-validated-input-component/src/index.jsx:12]: Nice! Just one suggestion:

isValid and warning stay in sync. If isValid is true then warning will be undefined, and if isValid is false then warning will be an error message.

You could get rid of isValid altogether by just using warning.

**Review by RobertGardner — APPROVED** [review]
Very well done. Just one minor suggestion.

File: react-validated-input-component/src/index.jsx (hunk 1 above)
Comment (joseph-w-valdez) [reply react-validated-input-component/src/index.jsx:12]: That's pretty smart. I'll make that change!

**Comment by joseph-w-valdez** [comment]
https://user-images.githubusercontent.com/117682160/217572612-e543097f-5cf4-41cc-a6cf-f130b5b00e31.mp4

**Pull request merged**